app/views/articles/_single_story.html.erb

```diff
@@ -48,6 +48,14 @@
           <a href="<%= story.path %>" class="crayons-story__tertiary fs-xs"><time datetime="<%= story.published_timestamp %>"><%= story.readable_publish_date %></time><span class="time-ago-indicator-initial-placeholder" data-seconds="<%= story.published_at_int %>"></span></a>
         </div>
       </div>
+
+      <% if defined?(pinned) && pinned == true %>
```

Comment:
Under which circumstances would the local variable `pinned` not exist?

Reply:
Love it! Didn't know about this injected variable :)